app/src/main/java/de/danoeh/antennapod/ui/screen/drawer/NavDrawerFragment.java

```diff
@@ -204,6 +208,26 @@ private boolean onTagContextMenuClicked(NavDrawerData.DrawerItem drawerItem, Men
         final int itemId = item.getItemId();
         if (itemId == R.id.rename_folder_item) {
             new RenameFeedDialog(getActivity(), drawerItem).show();
+            return true;
+        } else if (itemId == R.id.delete_folder_item) {
+            ConfirmationDialog dialog = new ConfirmationDialog(
+                    getContext(),
+                    R.string.delete_tag_label,
```

Comment:
Slightly more compact vertically:

```suggestion
                    getContext(), R.string.delete_tag_label,
```